code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -99,6 +99,9 @@
 	if(eaten_mob)
 		to_chat(src, span_warning("We cannot evolve with a belly full."))
 		return
+	
+	if(caste_type == /mob/living/carbon/xenomorph/hivemind && tgui_alert(src, "You are about to evolve into a hivemind, which places it's core on the tile you're on when evolving. This core cannot be moved and you cannot regress. Are you sure you would like to place your core here?", "Evolving to hivemind", list("Yes", "No"), FALSE) == "No")
```

Comment:
```suggestion
	if(caste_type == /mob/living/carbon/xenomorph/hivemind && tgui_alert(src, "You are about to evolve into a hivemind, which places its core on the tile you're on when evolving. This core cannot be moved and you cannot regress. Are you sure you would like to place your core here?", "Evolving to hivemind", list("Yes", "No"), FALSE) == "No")
```
'Its' is the possessive, 'it's' is the contraction.

Reply:
i hate english